config/nrfconnect/app/flashing.cmake

```diff
@@ -40,9 +40,9 @@ add_custom_command(OUTPUT "${FLASHBUNDLE_FLASHER_PLATFORM}"
     VERBATIM)
 
 if (merged_hex_to_flash)
-  set(flashbundle_hex_to_copy "zephyr/${merged_hex_to_flash}")
+  set(flashbundle_hex_to_copy "${merged_hex_to_flash}")
 else()
-  set(flashbundle_hex_to_copy "zephyr/${KERNEL_HEX_NAME}")
+  set(flashbundle_hex_to_copy "../merged.hex")
```

Comment:
I'm not 100%, as it's hard to find out in our new toolchain :) But I've tested builds without DFU and factory data and merged.hex file was always there (maybe it's because we always have sysbuild target now, so some sort of merging is always done).

Reply:
ok, thanks for checking!